test/vm-download

```diff
@@ -41,6 +41,12 @@ def download(link, force, stores):
         os.makedirs(DATA)
 
     dest = os.path.join(DATA, os.readlink(link))
+
+    # we have the file but there is not valid link
+    if os.path.exists(dest) and not os.path.exists(link):
+        os.symlink(os.path.exists(dest), os.readlink(link))
```

Comment:
> If not os.path.exists(link) then how does os.readlink(link) work?

While there was a copy/paste error and the link wasn't created in the right directory, here's the logic behind this: `os.path.exists(link)` will return `False` if the link is broken. `os.readlink` will read the content of the link, e.g. `openshift-c9cc3a6c15c001ff8de701f4dc6d68751f73cdf7.qcow2` (they way it's stored in our repo, as a relative link). Now if we have a $TEST_DATA/images directory with the right image but not a good link, then we create an image to that file in the images directory.
So this command will create a symlink like `openshift-c9cc3a6c15c001ff8de701f4dc6d68751f73cdf7.qcow2 -> /home/dev/cockpit-data/images/openshift-c9cc3a6c15c001ff8de701f4dc6d68751f73cdf7.qcow2` in the images directory, where it will be referenced by the corresponding git repo entry.